src/js/components/MenuComponent.jsx

```diff
@@ -0,0 +1,49 @@
+import React from "react/addons";
+
+import Util from "../helpers/Util";
+
+import MenuItemComponent from "../components/MenuItemComponent";
+
+var MenuComponent = React.createClass({
+  "displayName": "MenuComponent",
+
+  propTypes: {
+    children: Util.getComponentTypeValidator(MenuItemComponent),
```

Comment:
No, that one fails matching arrays of an instance and we want to match single instances as well as arrays of that instance.